modules/@ergonode/categories/src/components/Selects/CategorySelect.vue

```diff
@@ -393,6 +338,13 @@ export default {
 
             return [];
         },
+        visibleCategoryItems() {
+            if (!this.isCategoriesExpanded) {
+                return this.categoryItems.slice(0, 7);
```

Comment:
`7` should not be in the default variable?

Reply:
It's already `visibleCategoryItems` => 7 is pointing how many of them are sliced.